src/components/AnArticle.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import ReactHtmlParser from 'react-html-parser';
+
+const ratings = (aRating) => {
+  const stars = [];
+  const aStar = (<FontAwesomeIcon icon="star" />);
+  let counter = 0;
+  while (counter < aRating) {
+    stars.push(aStar);
+    counter += 1;
+  }
+  return (
+    <p>
+      {stars.map(item => <span className="text-warning">{item}</span>)}
+    </p>
+  );
+};
+
+const currentDate = (date) => {
+  const thisDate = new Date(date);
+  return thisDate.toDateString(date);
+};
+
+
+const AnArticle = (data) => {
```

Comment:
@Etomovich, it is better to pass props instead of a variable such as `data`, this makes the app more maintainable as it is easy to parse new props and add them to a destructure statement.

Reply:
Ok, I read an article that says components are to be stateless but containers should have state. But I will refactor this bit too

Reply:
Yes, but make sure you parse them in using props as the keyword.